src/main/java/Duke.java

```diff
@@ -7,7 +7,7 @@ public static void main(String[] args) {
 //                + "| |_| | |_| |   <  __/\n"
 //                + "|____/ \\__,_|_|\\_\\___|\n";
 //        System.out.println("Hello from\n" + logo);
-        String[] tasks = new String[100];
+        Task[] tasks = new Task[100];
```

Comment:
You are referring to Tasks here but I don't a Task class in your repository.
I think you forgot to track your files besides Duke.java on Git